core/lib/tasks/exchanges.rake

```diff
@@ -20,12 +20,13 @@ namespace :exchanges do
       begin
         item_charger.charge_for_items
       rescue Spree::UnreturnedItemCharger::ChargeFailure => e
-        failure = {message: e.message, new_order: e.new_order.try(:number)}
-      rescue Exception => e
-        failure = {message: "#{e.class}: #{e.message}"}
+        failure = { message: e.message }
+      rescue => e
```

Comment:
I'm fine with rescuing Exception as long as we re-raise it. It's just not one of those things that should be swallowed.